strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/NugetGroupRepositoryProviderTest.java

```diff
@@ -120,29 +121,27 @@ public void setUp()
         repositoryGroup.addRepositoryToGroup(REPOSITORY_RELEASES_2);
         repositoryGroup.addRepositoryToGroup(REPOSITORY_RELEASES_3);
 
-        createRepository(repositoryGroup);
+        createRepository(STORAGE0, repositoryGroup);
 
-        Repository repositoryWithNestedGroupLevel1 = nugetRepositoryFactory.createRepository(STORAGE0,
-                                                                                             REPOSITORY_GROUP_WITH_NESTED_GROUP_1);
+        MutableRepository repositoryWithNestedGroupLevel1 = nugetRepositoryFactory.createRepository(REPOSITORY_GROUP_WITH_NESTED_GROUP_1);
         repositoryWithNestedGroupLevel1.setType(RepositoryTypeEnum.GROUP.getType());
         repositoryWithNestedGroupLevel1.setAllowsRedeployment(false);
         repositoryWithNestedGroupLevel1.setAllowsDelete(false);
         repositoryWithNestedGroupLevel1.setAllowsForceDeletion(false);
         repositoryWithNestedGroupLevel1.setRepositoryConfiguration(nugetRepositoryConfiguration);
         repositoryWithNestedGroupLevel1.addRepositoryToGroup(REPOSITORY_GROUP);
 
-        createRepository(repositoryWithNestedGroupLevel1);
+        createRepository(STORAGE0, repositoryWithNestedGroupLevel1);
 
-        Repository repositoryWithNestedGroupLevel2 = nugetRepositoryFactory.createRepository(STORAGE0,
-                                                                                             REPOSITORY_GROUP_WITH_NESTED_GROUP_2);
+        MutableRepository repositoryWithNestedGroupLevel2 = nugetRepositoryFactory.createRepository(REPOSITORY_GROUP_WITH_NESTED_GROUP_2);
         repositoryWithNestedGroupLevel2.setType(RepositoryTypeEnum.GROUP.getType());
         repositoryWithNestedGroupLevel2.setAllowsRedeployment(false);
         repositoryWithNestedGroupLevel2.setAllowsDelete(false);
         repositoryWithNestedGroupLevel2.setAllowsForceDeletion(false);
         repositoryWithNestedGroupLevel2.setRepositoryConfiguration(nugetRepositoryConfiguration);
         repositoryWithNestedGroupLevel2.addRepositoryToGroup(REPOSITORY_GROUP_WITH_NESTED_GROUP_1);
 
-        createRepository(repositoryWithNestedGroupLevel2);
+        createRepository(STORAGE0, repositoryWithNestedGroupLevel2);
```

Comment:
You see what I mean? Here it's the other (proper) way around. Consistency saves wasting time in unnecessary confusion. :)